frontend/src/app/modules/home/modals/dashboard/dashboard-template.html

```diff
@@ -1,39 +1,37 @@
-<p-dialog [visible]="true" [draggable]="false" [resizable]="false" [modal]="true">
+<div class="modal-container">
     <form [formGroup]="formGroup" (ngSubmit)="saveHandle()">
-        <p-panel header={{title}}>
-            <div class="input-section">
-                <span class="p-float-label">
-                    <input id="float-input-dashboard-name" type="text" pInputText formControlName="name">
-                    <label for="float-input-dashboard-name">Dashboard name</label>
-                </span>
-                <div *ngIf="formGroup.controls.name.touched && formGroup.controls.name.invalid" class="validation">
-                    <small *ngIf="formGroup.controls.name.errors.required">Field name can't be empty</small>
-                    <small *ngIf="formGroup.controls.name.value.length < 3">Min name length 3</small>
-                    <small *ngIf="formGroup.controls.name.value.length > 50">Max name length 50</small>
-                    <small *ngIf="formGroup.controls.name.errors?.notUnique">Dashboard name is not unique</small>
-                    <small *ngIf="formGroup.controls.name.hasError('pattern')">Name must contain latin letters, numbers,
-                        underscores, dashes, dots or spaces</small>
-                </div>
+        <div class="input-section">
+            <span class="p-float-label">
+                <input id="float-input-dashboard-name" type="text" pInputText formControlName="name">
+                <label for="float-input-dashboard-name">Dashboard name</label>
+            </span>
+            <div *ngIf="formGroup.controls.name.touched && formGroup.controls.name.invalid" class="validation">
+                <small *ngIf="formGroup.controls.name.errors.required">Field name can't be empty</small>
+                <small *ngIf="formGroup.controls.name.value.length < 3">Min name length 3</small>
+                <small *ngIf="formGroup.controls.name.value.length > 50">Max name length 50</small>
+                <small *ngIf="formGroup.controls.name.errors?.notUnique">Dashboard name is not unique</small>
+                <small *ngIf="formGroup.controls.name.hasError('pattern')">Name must contain latin letters, numbers,
+                    underscores, dashes, dots or spaces</small>
```

Comment:
Can you please rewrite this code with using get and set, you can see examples in  organization settings component or create organization components. Also, can you please rewrite hasError('pattern') to errors?.pattern